fdk/customer_code.py

```diff
@@ -24,6 +22,7 @@ def __init__(self, func_module_path, entrypoint="handler"):
 
     @staticmethod
     def import_from_source(func_module_path):
+        from importlib import util
```

Comment:
trying to reduce imports time by delaying code initialization as close as possible to the time of the actual call.